### ISSUE TITLE
Gupplyplex. illegal instruction, core dump

### ISSUE THREAD

**Issue by omarkr8** [Psy-Fer/interARTIC#69]
Finding strange output at the first step of the pipeline, im assuming it is trying to use guppyplex to collect demux data.

output looks like 
***RUNNING GUPPYPLEX COMMAND***
Illegal instruction (core dump)

and that is repeated many many times, presumably for every barcode.

I do not believe this is an issue with the pipeline per se, because i can run the same parameters and data on a different machine with no obvious issues at this step. 
So the question is, what is it with my VMM Ubuntu setup on this problematic machine, that is different to the machine that worked (I have no idea what parts to look at that could be important).

the reason im using this problematic machine is because its meant to have a more powerful CPU, so i just hope this core dump issue is just a matter to reallocating resources.

**Comment by hasindu2008** [comment]
This kind of error was observed in ARM-v8 processors due to openBLAS core type not being recognised correctly, which was then fixed. 
Is your processor Intel or ARM? Can you send the details of your processor  - output of cat /proc/cpuinfo?

**Comment by omarkr8** [comment]
So i've given Ubuntu 3 processors using VMM...


processor	: 0
vendor_id	: GenuineIntel
cpu family	: 6
model		: 85
model name	: Intel(R) Xeon(R) Silver 4208 CPU @ 2.10GHz
stepping	: 7
microcode	: 0xffffffff
cpu MHz		: 2095.078
cache size	: 11264 KB
physical id	: 0
siblings	: 3
core id		: 0
cpu cores	: 3
apicid		: 0
initial apicid	: 0
fpu		: yes
fpu_exception	: yes
cpuid level	: 22
wp		: yes
flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx rdtscp lm constant_tsc rep_good nopl xtopology nonstop_tsc cpuid tsc_known_freq pni ssse3 cx16 pcid sse4_1 sse4_2 hypervisor lahf_lm invpcid_single ibrs_enhanced fsgsbase invpcid md_clear flush_l1d arch_capabilities
bugs		: spectre_v1 spectre_v2 spec_store_bypass swapgs itlb_multihit
bogomips	: 4190.15
clflush size	: 64
cache_alignment	: 64
address sizes	: 46 bits physical, 48 bits virtual
power management:

processor	: 1
vendor_id	: GenuineIntel
cpu family	: 6
model		: 85
model name	: Intel(R) Xeon(R) Silver 4208 CPU @ 2.10GHz
stepping	: 7
microcode	: 0xffffffff
cpu MHz		: 2095.078
cache size	: 11264 KB
physical id	: 0
siblings	: 3
core id		: 1
cpu cores	: 3
apicid		: 1
initial apicid	: 1
fpu		: yes
fpu_exception	: yes
cpuid level	: 22
wp		: yes
flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx rdtscp lm constant_tsc rep_good nopl xtopology nonstop_tsc cpuid tsc_known_freq pni ssse3 cx16 pcid sse4_1 sse4_2 hypervisor lahf_lm invpcid_single ibrs_enhanced fsgsbase invpcid md_clear flush_l1d arch_capabilities
bugs		: spectre_v1 spectre_v2 spec_store_bypass swapgs itlb_multihit
bogomips	: 4190.15
clflush size	: 64
cache_alignment	: 64
address sizes	: 46 bits physical, 48 bits virtual
power management:

processor	: 2
vendor_id	: GenuineIntel
cpu family	: 6
model		: 85
model name	: Intel(R) Xeon(R) Silver 4208 CPU @ 2.10GHz
stepping	: 7
microcode	: 0xffffffff
cpu MHz		: 2095.078
cache size	: 11264 KB
physical id	: 0
siblings	: 3
core id		: 2
cpu cores	: 3
apicid		: 2
initial apicid	: 2
fpu		: yes
fpu_exception	: yes
cpuid level	: 22
wp		: yes
flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx rdtscp lm constant_tsc rep_good nopl xtopology nonstop_tsc cpuid tsc_known_freq pni ssse3 cx16 pcid sse4_1 sse4_2 hypervisor lahf_lm invpcid_single ibrs_enhanced fsgsbase invpcid md_clear flush_l1d arch_capabilities
bugs		: spectre_v1 spectre_v2 spec_store_bypass swapgs itlb_multihit
bogomips	: 4190.15
clflush size	: 64
cache_alignment	: 64
address sizes	: 46 bits physical, 48 bits virtual
power management:

**Comment by omarkr8** [comment]
the cat output was in the VMM
im using Oracle VM virtualbox Manager

host machine is a Windows 10 Pro.
I cant cat/proc on command line. perhaps an alternate command "wmic cpu" gives me

Intel64 Family  6 Model 85 Stepping 7
Intel(R) Xeon (R) Silver 4208 CPU@ 2.10GHz
8 Cores

**Comment by hasindu2008** [comment]
Right, seems like  Oracle VM VirtualBox Manager is not exposing AVX and this might be the cause https://stackoverflow.com/questions/65780506/how-to-enable-avx-avx2-in-virtualbox-6-1-16-with-ubuntu-20-04-64bit. 

However, if you are running on Windows 10, there is another easy way to run interARTIC without any VMs - natively on Windows using WSL. Have you got WSL installed - if not it is pretty straight forward https://www.windowscentral.com/install-windows-subsystem-linux-windows-10. This is the method I use to run interARTIC on my Windows laptop.

**Comment by hasindu2008** [comment]
Perhaps Could you try this https://petri.com/how-to-disable-hyper-v-completely-in-windows-10?
